### PR TITLE
#6934 update framework to multiple environments

### PR DESCRIPTION
**Pull request by rdutu-vg**
Draft PR as backup during refactorization

### CODE REVIEW

File: sormas-e2e-tests/scripts/runTests.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ rm -rf ./allureReports
 echo "Executing gradle clean..."
 ./gradlew clean goJF
 echo "Starting all BDD tests under @$1 tag..."
-./gradlew startTests -Dcucumber.tags=\"@$1\" -Dheadless=true -Dcourgette.threads=9
+./gradlew startTests -Dcucumber.tags=\"@$1\" -Dheadless=true -Dcourgette.threads=9 -DenvConfig=C:/Users/Zack/Desktop/data.json
```
Comment (mario-kuehne) [inline sormas-e2e-tests/scripts/runTests.sh:28]: envConfig could be parameterized

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/envconfig/configprovider/ConfigFileReader.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
```
Comment (mario-kuehne) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/envconfig/configprovider/ConfigFileReader.java:3]: `2016-2021` -> `2016-2022`

Comment (mario-kuehne) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/envconfig/configprovider/ConfigFileReader.java:3]: see also other newly introduced files

File: sormas-e2e-tests/scripts/runTests.sh (hunk 1 above)
Comment (rdutu-vg) [reply sormas-e2e-tests/scripts/runTests.sh:28]: yes, this was commited by mistake to check the regression, we'll hardcode this path into jenkins machine once done

**Comment by rdutu-vg** [comment]
Local execution

local execution

Jenkins last execution

last jenkins execution

**Comment by rdutu-vg** [comment]
checks

Implementation is working as expected.

**Comment by rdutu-vg** [comment]
https://jenkins.sormas.netzlink.com/view/SORMAS-Project/job/sormas-RunE2eTests-branch/11/

**Pull request merged**